### PR TITLE
[PWGLF] Update NucleiHistTask.cxx

### PR DESCRIPTION
**Pull request by RafRaf11**
Added new configurable in order to allow for different momentum scalings for Z=2 species.

### CODE REVIEW

File: PWGLF/Tasks/Nuspex/NucleiHistTask.cxx [hunk 1]
```diff
@@ -507,8 +543,8 @@ struct NucleiHistTask {
         proton_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaProton);
         deuteron_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaDeut);
         triton_reg.fill(HIST("histTpcNsigmaData"), momentum, nSigmaTriton);
-        Helium3_reg.fill(HIST("histTpcNsigmaData"), momentum * 2.0, nSigmaHe3);
-        Helium4_reg.fill(HIST("histTpcNsigmaData"), momentum * 2.0, nSigmaHe4);
+        Helium3_reg.fill(HIST("histTpcNsigmaData"), momentum_Z_equal_2 * 2.0, nSigmaHe3);
```
Comment (maciacco) [inline PWGLF/Tasks/Nuspex/NucleiHistTask.cxx:546]: Just a small doubt before merging: is this multiplication by 2 still needed now that you are already multiplying in the switch-case above?

**Review by maciacco — COMMENTED** [review]
please just check above before merging

File: PWGLF/Tasks/Nuspex/NucleiHistTask.cxx (hunk 1 above)
Comment (RafRaf11) [reply PWGLF/Tasks/Nuspex/NucleiHistTask.cxx:546]: Thanks for noticing, I overlooked it.

**Review by maciacco — APPROVED** [review]

**Comment by RafRaf11** [comment]
Hello, is it possible to re-test the build check? It seems not to be the PR's fault that it failed

**Comment by maciacco** [comment]
hello @RafRaf11 maybe you could try to close and reopen the PR. However I think there are still some issues with the o2 compilation check due to insufficient space on the testing machine according to the logs of other PRs

**Pull request closed without merging**